client/src/components/App.jsx

```diff
@@ -1,23 +1,60 @@
 import React from 'react';
 import ReactDom from 'react-dom';
 import ListingDetail from './ListingDetail.jsx';
+import axios from 'axios';
+const Promise = require('bluebird');
+
+const API_URL = 'http://localhost:3005';
```

Comment:
I haven't really used axios, so is it possible to use relative urls, instead of hardcoding the api_url?